app/controllers/autocomplete_controller.rb

```diff
@@ -138,7 +138,7 @@ def external_work
 
   # encodings for importing
   def encoding
-    encodings = Encoding.name_list.select {|e| e.match(/#{params[:term]}/i)}
+    encodings = Encoding.name_list.select { |e| e.match(/#{Regexp.escape(params[:term])}/i) }
     render_output(encodings)
   end
```

Comment:
Let's delete this instead, since we don't have an autocomplete for import encodings.